SpeechTranslator/CS/ConstantParam.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace SpeechTranslator
+{
+    class ConstantParam
+    {           
+        //Token Property
+        public static string subscriptionKey = "47fe38401a6e45699b1ff76c8d3681d1";

```

Comment:
Revert this back to         

public static string subscriptionKey = "****REPLACE THIS WITH YOUR AZURE COGNITIVE SERVICES SUBSCRIPTION KEY****";

Don't commit private subscription keys.

Reply:
Done the changes.